model/bitmap-constructor.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const Bitmap = module.exports = function Bitmap(buffer, callback){
+  this.size = buffer.readInt32LE(2);
+  this.imageStart = buffer.readInt32LE(10);
+  this.headerSize = buffer.readInt32LE(14);
+  this.imageSize = buffer.readInt32LE(34);
+  this.numColors = buffer.readInt32LE(46);
+  this.head = buffer.toString('hex', 0, 14 + this.headerSize);
+  let bitmapColorArray = [];
+  let colorMapStart = 14 + this.headerSize;
+  let singleColorStart = colorMapStart;
+  for(let i = 0; i < this.numColors; i++){
+    let singleColorArray = [];
+    for(let j = singleColorStart; j < (singleColorStart + 4); j++){
+      singleColorArray.push(buffer.readUInt8(j));
+    }
+    bitmapColorArray.push(singleColorArray);
+    singleColorStart += 4;
+  }
+  this.color = bitmapColorArray;
+  this.pixel = buffer.toString('hex', this.imageStart, this.imageStart + this.imageSize);
+  if(callback) callback(null, this);
+};
```

Comment:
@brials @remilonwheels @caylazabel `Bitmap` constructor looks good - may be useful to also house the raw `buffer` as part of the constructor for later use